README.md

```diff
@@ -15,12 +15,20 @@ To ensure the long-term health of the project, we are temporarily pausing all of
 
 ✅ We will continue reviewing and merging community PRs.
 
+✅ No CI Changes at all - Linting, Testing, e2e testing are all happening on every single pull request.
+
+✅ Security updates, patches, etc. will still be merged to the project (to the `main` branch)
+
 ✅ Contributions will be available on the main branch.
 
 ❌ We will not provide support via GitHub Discussions, Slack, or issue comments.
 
 ❌ We will not publish any new releases (major, minor, or patch), including 0.19.x and 1.0.x.
```

Comment:
Did you mean "We will not publish any new github releases (major, minor, or patch), regardless of the branch or version number." ?

Reply:
We are still pushing releases to the `main` branch - but I'll review to be clearer

Reply:
Thanks. So you keep releases without backports? In that case what do you produce as artifacts? Just an archive matching the tag of the code?

Reply:
>  So you keep releases without backports? In that case what do you produce as artifacts?

This is not even part of htis move - we don't do it ever since 0.14 - https://external-secrets.io/latest/introduction/stability-support/

Our artifacts are source code, packages, helm charts, and docker images. We need to tag them because once that old release was our newest shiniest brightest.